jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js

```diff
@@ -347,6 +350,9 @@ var module = module;
         }, this);
       }
       if (processed.entityType === 'bundle') {
+        if (meta['bundle-unit-count']) {
```

Comment:
Nothing is displayed. That causes problems with our layout, but I think that's more a function of CSS than JS (i.e., our table cells need to remain the same width even when they don't have content).